src/pytorch_lightning/plugins/precision/deepspeed.py

```diff
@@ -88,7 +88,7 @@ def _run_backward(
 
     def optimizer_step(
         self,
-        model: Union["pl.LightningModule", Module],
+        model: Optional[Union["pl.LightningModule", Module]],
```

Comment:
```suggestion
        model: Union["pl.LightningModule", Module, None],
```
Maybe, fewer brackets?

Reply:
I myself am fine with both and find it a matter of preference.

Reply:
I would stick with `Optional` as this is what we have in other places as well :)